yxi/axi-calyx/cocotb/axi_test.py

```diff
@@ -78,11 +87,21 @@ async def run_kernel_test(toplevel, data_path: str):
     await Timer(100, "ns")
     await FallingEdge(toplevel.clk)
 
-
     # Finish when ap_done is high or 100 us of simulation have passed.
     timeout = 5000
+    #Base addresses for memories
+    await tb.control_manager.write(0x0010, encode([0x0],4))
+    await tb.control_manager.write(0x0014, encode([0x0],4))
+    await tb.control_manager.write(0x0018, encode([0x0],4))
+    await tb.control_manager.write(0x001C, encode([0x0],4))
+    await tb.control_manager.write(0x0020, encode([0x0],4))
+    await tb.control_manager.write(0x0024, encode([0x0],4))
+    #Assert ap_start by writing 1 to 0x0000
+    await tb.control_manager.write(0x0000, encode([0x1],1))
```

Comment:
TODO: Change this from being hardcoded to be dynamic and also dependent on if we are targeting Xilinx or not